index.js

```diff
@@ -3,30 +3,84 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+
+let flag = true
+if(rolls.length === 0) {
+  return true
+}
+  for (let i = 0; i < rolls.length; i++) {
+    if(typeof rolls[i] === 'number') {
+      return flag
+    } 
+    flag = false
```

Comment:
we only want to reassign `flag` to false if the element is not a number. Reconsider your conditional on line 13.